src/com/cft/contactmerge/MapMaker.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * MapMaker is responsible for creating the map file that tells ContactImporter
+ * and MergeFileExporter how to map properties (FirstName, Address, etc) to
+ * the columns in the file.
+ */
+package com.cft.contactmerge;
+
+import java.util.*;
+
+public class MapMaker {
+    public void addColumnMap(String property, int columnInFile)
+    {
+        // TODO: Implementation needed
+    }
+
+    public void removeColumnMap(String propertyName)
+    {
+        // TODO: Implementation needed
+    }
+
+    public void addStaticMap(String property, String value)
```

Comment:
Not sure why this will be needed?

Reply:
This is specifically for ITK use cases. When you import phone numbers and e-mails you need to indicate which type of phone/email. Typically our input data has one column per phone/email type. We have to add a column for each of this columns to indicate the type name.  So... "MobilePhone, HomePhone" would turn into "MobilePhoneName, MobilePhone, HomePhoneName, HomePhone". Sample data for the second would be "Mobile Phone, (617) 571-3851, Home Phone, (520) 555-1423". The value of MobilePhoneName and HomePhoneName is always the same.